shared/src/commonTest/kotlin/com.kcc.kmmhackathon.shared.utility/extensions/IntExtensionsTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.kcc.kmmhackathon.shared.utility.extensions
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class IntExtensionsTest {
+
+    @Test
+    fun testInt_getPortionsString_expectedValues() {
+        val loadsAvailable = 200
+        val oneAvailable = 1
+        val noneAvailable = 0
+
+        assertEquals(loadsAvailable.getPortionsString(), "200 portions remaining")
+        assertEquals(oneAvailable.getPortionsString(), "1 portion remaining")
+        assertEquals(noneAvailable.getPortionsString(), "Reserved")
+    }
+
+    @Test
+    fun testInt_getPortionsString_negativeValues() {
+        val negativeOneAvailable = -1
+        val negativeLoadsAvailable = -200
+
+        assertEquals(negativeOneAvailable.getPortionsString(), "Reserved")
+        assertEquals(negativeLoadsAvailable.getPortionsString(), "Reserved")
+    }
```

Comment:
Is this test necessary? Do we have the case where the portion numbers can be below 0?

Reply:
You can't reserve (reduce the portion # by 1) once the quantity is 0 because the button is disabled, right?

Reply:
It was there for completeness, but as noted by both you and Gareth, there should be no case where the quantity is < 0